tools/infer/predict_det.py

```diff
@@ -32,11 +32,9 @@
 from ppocr.postprocess import build_post_process
 import json
 
-logger = get_logger()
-
 
 class TextDetector(object):
-    def __init__(self, args):
+    def __init__(self, args, logger):
```

Comment:
logger arg needs a default value None, the default behavior should be using logger from `logger = get_logger()`.

otherwise, it will break this: https://github.com/PaddlePaddle/PaddleOCR/blob/1569cfb430dee06b23413b7b0a6d99dc8c84ab70/tools/infer/predict_system.py#L51